shared/src/business/useCases/featureFlag/getAllFeatureFlagsInteractor.ts

```diff
@@ -0,0 +1,34 @@
+import { ALLOWLIST_FEATURE_FLAGS } from '../../entities/EntityConstants';
+
+/**
+ * getAllFeatureFlagsInteractor
+ *
+ * @param {object} applicationContext the application context
+ * @param {string} providers.featureFlag the feature flag to get the value for
+ * @returns {boolean} true if result of the persistence method is 'true'; false otherwise
```

Comment:
🤔 hm, wouldn't this return an `object` instead of a `boolean`? Maybe I'm missing something.